src/Kros.AspNetCore/Authorization/GatewayJwtAuthorizationOptions.cs

```diff
@@ -53,8 +53,13 @@ private string GetUrl(IServiceDiscoveryProvider provider, AuthorizationServiceOp
         public TimeSpan CacheSlidingExpirationOffset { get; set; } = TimeSpan.Zero;
 
         /// <summary>
-        /// Paths that do not use JWT authorization caching
+        /// Paths that do not use JWT authorization caching.
         /// </summary>
         public List<string> IgnoredPathForCache { get; private set; } = new List<string>();
+
+        /// <summary>
+        /// Headers wich if are in request, will be forwarded to client.
+        /// </summary>
+        public List<string> ForwardedHeaders { get; private set; } = new List<string>();
```

Comment:
Nemusíš to meniť, ale keď sa to nastavuje takto, tak tam netreba dávať ten privátny `set`. Stačí:

``` c#
public List<string> ForwardedHeaders { get; } = new List<string>();
```

Technicky to berie ako keby nastavenie na začiatku konštruktora.

Ale teoreticky je možné, že ten privátny set potrebuje keď to napĺňa automaticky z `appsettings.json`, ale nechce sa mi to overovať. :)